packages/core-mobile/app/navigation/types.ts

```diff
@@ -132,13 +137,15 @@ export type ApprovalPopupParams = {
     account,
     maxFeePerGas,
     maxPriorityFeePerGas,
-    overrideData
+    overrideData,
+    isTestnet
   }: {
     network: Network
     account: CorePrimaryAccount
     maxFeePerGas?: bigint
     maxPriorityFeePerGas?: bigint
     overrideData?: string
+    isTestnet?: boolean
```

Comment:
Ray removed isTestnet param btw. we are just using network.isTestnet now.

Reply:
removed again.